anatomy/tracks/config-json.md

```diff
@@ -60,6 +60,7 @@ Each concept exercise is an entry in the `exercises.concept` array. The followin
 - `name`: the exercise's name
 - `concepts`: an array of concept slugs that are taught by this concept exercise
 - `prerequisites`: an array of concept slugs that must be unlocked before a student can start this exercise
+- `status` (optional): the exercise's status, which is either `"active"`, `"wip"`, `"beta"` or `"deprecated"`; defaults to `"active"` if not specified
 
```

Comment:
```suggestion
  - `wip`: A work-in-progress exercise not ready for public consumption. Exercises with this tag will not be shown to students on the UI or be used for unlocking logic. They may appear for maintainers.
  - `beta`: This signifies active exercises that are new and which we would like feedback on. We show a beta label on the site for these exercise, with a CTA of “Please give us feedback”
  - `active`: The normal state of active exercises
  - `deprecated`: Old exercises to be removed (not usable at this stage).

```

If/when you're happy with this it needs copying down to Practice Exercises below.